src/main/java/org/folio/rest/impl/EholdingsProvidersImpl.java

```diff
@@ -130,10 +138,39 @@ public void getEholdingsProvidersByProviderId(String providerId, String include,
 
   @Override
   @HandleValidationErrors
-  public void putEholdingsProvidersByProviderId(String providerId, String contentType, ProviderPutRequest entity, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-    asyncResultHandler.handle(Future.succeededFuture(GetEholdingsProvidersResponse.status(Response.Status.NOT_IMPLEMENTED).build()));
+  public void putEholdingsProvidersByProviderId(String providerId, String contentType, ProviderPutRequest entity,
```

Comment:
could you please add a content-type validation as it is for Ruby code
<img width="1183" alt="screen shot 2018-11-12 at 12 14 03 pm" src="https://user-images.githubusercontent.com/37537790/48340939-7cd92300-e674-11e8-93ad-e36d3df1f618.png">

Reply:
From my testing with either Postman or Advanced REST Client I am currently getting an error response  HTTP 400 with body similar to `Content-type header must be ["application/vnd.api+json"] but it is "application/json"` for either an empty or incorrect content type. So I believe this case is being handled by framework 
![screen shot 2018-11-12 at 2 02 27 pm](https://user-images.githubusercontent.com/19415226/48369177-c9782a80-e683-11e8-9078-6b273419117d.png)